docs/get_started/raspbian_setup.md

```diff
@@ -1,7 +1,7 @@
 # Installing MXNet on Raspbian
-MXNet currently supports the Debian based Raspbian operating system for Raspberry Pi Devices, offering a python API. 
+MXNet currently supports a python API for the Debian based Raspbian operating system for Raspberry Pi Devices.
 
-The full MxNet library is over 200MB when loaded into memory and the requirements can take almost 1GB of disk space. Due to this we currently reccomend running MXNet on the Raspberry Pi 3 or equivilant devices with more than 1GB of RAM and with an SD card that has at least 4 GB of memory free. The Raspberry Pi 1, 2, Zero and other devices with less than 1GB of RAM are not sufficient to run the full MXNet library (though they can run the MXNet amalgamation library). 
+The full MxNet library is over 200MB when loaded into memory and the requirements can take almost 1GB of disk space. Due to the size we currently reccomend running MXNet on the Raspberry Pi 3 or equivilant devices with more than 1GB of RAM and with an SD card that has at least 4 GB of memory free. The Raspberry Pi 1, 2, Zero and other devices with less than 1GB of RAM are not sufficient to run the full MXNet library (though they can run the MXNet amalgamation library). 
```

Comment:
The complete MXNet library exceeds 200 MB and the requirements can take almost 1 GB of disk space. Because of this, we recommend running MXNet on the Raspberry Pi 3 or an equivalent device that has more than 1 GB of RAM and a Secure Digital (SD) card that has at least 4 GB of free memory. The Raspberry Pi 1, 2, Zero and other devices with less than 1 GB of RAM cannot run the compete MXNet library.